app/assets/stylesheets/orgchart.scss

```diff
@@ -0,0 +1,141 @@
+.orgchart_menu {
+  width: 120px;
+  position:absolute;
+  padding: 0;
+  margin: 0;
+  
+  li {
+    a {
+      padding: 4px 10px;
+      display: block;
+    }
+  }
+  
+  li.active {
+    font-weight: bold;
+    background: #ccc;
+  }
+}
+
+@media print {
+  .orgchart_menu {
+    display:none;
```

Comment:
feels a bit weird to put this media query inside a css file explicitly marked for `screen`, but I gather the orgchart page was going through hoops to remain print-friendly and thought I should carry that tradition forward. Everything is compiled into application.css with a `require_tree .` so I didnt have much flexibility to spin up a new print stylesheet without re-arranging a bunch of stuff.